graveler/ref/iterator.go

```diff
@@ -1,10 +1,14 @@
 package ref
 
+import "errors"
+
 const (
 	iteratorOffsetGE = ">="
 	iteratorOffsetGT = ">"
 )
 
+var ErrIteratorClosed = errors.New("iterator already closed")
```

Comment:
All of these should be the same error.

Also: what is The Go Dan to keep these changes sync'ed?  I'm reviewing these by reading through them, but this is a *horrible* way to do it.  Surely we're not the first people to run into this...

Reply:
https://github.com/googleapis/google-api-go-client/blob/v0.36.0/iterator/iterator.go

can modify the code not to panic - just report an error. can condition the SeekGE not to reset the error just for this case / state.

What do you think?

Reply:
Yeah.  Probably not in this PR, though :-)

Reply:
Don't stop - we will get it better next round